client/js/table-view.js

```diff
@@ -22,6 +22,7 @@ class TableView {
 	initDomReferences() {
 		this.headerRowEl = document.querySelector('THEAD TR');
 		this.sheetBodyEl = document.querySelector('TBODY');
+		this.sheetFootEl = document.querySelector('TFOOT TR');
```

Comment:
The terminology that an HTML table uses is "header" and "footer". So that other devs know immediately what this element is, change the variable to "footerRowEl"; it's basically the same thing as the "headerRowEl" but for the footer.